modules/oe_theme_content_call_tenders/templates/oe-theme-content-call-tenders-label-status.html.twig

```diff
@@ -12,7 +12,7 @@
   {% elseif name == 'open' %}
     {% set background_class = ' ecl-label--high' %}
   {% elseif name == 'closed' %}
-    {% set background_class = ' ecl-label--highlight' %}
+    {% set background_class = ' ecl-label--low' %}
```

Comment:
Class ecl-label--low defines background colour, but as I see on the page https://webgate.ec.europa.eu/fpfis/wikis/display/EWPP/Call+for+Tenders+Content+Type+Specification+-+Version+1.0.0?preview=/437857073/451122351/Teaser%20view.png text for closed status has to be black.
<img width="580" alt="Screenshot 2020-10-05 at 12 04 29" src="https://user-images.githubusercontent.com/1183951/95060596-28753b80-0703-11eb-9b97-c0275fbaf008.png">

Reply:
The same happens on the other status cases, i change the text color to black on all of them:
<img width="672" alt="status_black" src="https://user-images.githubusercontent.com/5982573/95067487-35def580-0704-11eb-8e02-04694a95ae16.png">